### PR TITLE
To Do List Structure

### PR DESCRIPTION
**Pull request by Yulianav09**
# To-Do List Structure

- Used Webpack to build the project.
- Used  HTML Webpack Plugin.
- Created an array to contain some simple tasks.
- HTML is populated automatically.
- CSS Styles are loaded using the Webpack Style/CSS Loader.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,171 @@
...
+- **Saved tasks when you refresh**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yulianav09.github.io/To-Do-list/)
+
```
Comment (Efiamotu-1) [inline README.md]: - [x] This link provided here doesn't display the live version of your todo list project. I kindly suggest you provide the appropriate link or remove this section if the link is not yet available 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,171 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@gYuliana09](https://github.com/Yulianav09)
+- Twitter: [@Yuliana Velez Rodríguez](https://twitter.com/YulianaVel09e)
```
Comment (Efiamotu-1) [inline README.md]: - [x] The link you provided for twitter is broken and currently doesn't lead to an existing profile. I kindly suggest you open an account if you do not have one currently or fix the appropriate link 👍

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Yulianav09 , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Good job setting up webpack correctly
- ✔️ Great UI design
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Efiamotu-1** [comment]
- [x] kindly provide adequate steps and information on how to install your project. see below the provided example 👍 
- example
- > After installing the project, run the following command:
- run `npm i` or `npm install` in your terminal to download all the packages needed to make the project run
- run `npm run start` or `npm start` to open the project in your browser 
- * This is all that should be required of you to assist anyone using your app for the first time whether it is a developer or just a random user 👍

File: README.md (hunk 1 above)
Comment (Yulianav09) [reply README.md]: Thanks for the suggestion, I removed the link as we currently do not know how to deploy this project using Webpack, the normal process we used for previous projects seems to be not working with Webpack 😄

File: README.md (hunk 2 above)
Comment (Yulianav09) [reply README.md]: Thanks for pointing this out, I fixed the link

**Review by Davidosky007 — APPROVED** [review]
Hi @Yulianav09 👋 ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Webpack setup properly 👍 
* Good looking readme 👍 
* Well written PR 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**